app/models/collection.rb

```diff
@@ -6,9 +6,12 @@ class Collection < ApplicationRecord
   belongs_to :user
   has_many :works, dependent: :destroy
 
-  has_and_belongs_to_many :depositors, class_name: 'User', join_table: 'depositors'
-  has_and_belongs_to_many :managers, class_name: 'User', join_table: 'managers'
-  has_and_belongs_to_many :reviewers, class_name: 'User', join_table: 'reviewers'
+  has_many :collection_depositor, dependent: :destroy
+  has_many :collection_manager, dependent: :destroy
```

Comment:
I was surprised to see two `has_many` relationships expressed as singular nouns instead of plural ones. I'm assuming there's a reason for them being singular despite the convention?

Reply:
Good catch. Fixed.